site/docs/v1/tech/reference/cookies.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: doc
+title: Hosted Login Pages Cookies
+description: A list of cookies set by the FusionAuth Hosted Login pages
+navcategory: admin
+---
+
+== Overview
+
+Cookies are a critical part of web applications, and FusionAuth sets cookies. When you call certain APIs, such as the Login API, cookies may be set. These cookies are specified in the link:/docs/v1/tech/apis[API documentation].
+
+When you use the link:/docs/v1/tech/core-concepts/integration-points#hosted-login-pages[hosted login pages], FusionAuth sets cookies to enable functionality.
+
+== Domains
+
+The domain of all cookies is limited to the domain on which the FusionAuth instance is running. You can control the domain FusionAuth uses by link:/docs/v1/tech/admin-guide/proxy-setup[setting up a proxy].
+
+In other words, if FusionAuth lives at `auth.piedpiper.com`, it will only set cookies for that domain. It will never set cookies for `.piedpiper.com`.
+
+== Cookie List
+
+[WARNING.warning]
+====
+Cookies set by the hosted login pages are displayed here for informational purposes only.
+
+Duration, name, and purpose are subject to change.
+====
+
+When a cookie has a value of `Yes` for the `Prefix` column, name is used as a prefix for one or more cookies.
+
+//Pulled from Cookies.java
+
+.Hosted Login Page Cookies 
+[cols="2,1,1,3"]
+|===
+| Name | Prefix | Duration | Function
+
+| fusionauth.csrf
+| No
+| Session
+| Protects against Cross Site Request Forgery.
+
+| fusionauth.flash-message
+| No
+| Session
+| Displays information or error messages.
+
+| fusionauth.known-device
+| Yes
+| 70 Years
+| Determines if this device is known.
+
+| fusionauth.locale
+| No
+| 70 Years
+| The locale of the user.
+
+| fusionauth.app.pkce-verifier
+| No
+| 70 Years
+| Security measures around login.
+
+| fusionauth.pkce-verifier
+| No
+| Session
+| Security measures around login.
+
+| fusionauth.remember-device
+| No
+| 70 years
+| Records if the user wants to remain logged in on this device.
+
+| fusionauth.sso
+| No
+| 70 Years
+| Implements single sign-on functionality.
+
+| fusionauth.timezone
+| No
+| Session
+| The timezone of the user.
+
+| fusionauth.trusted-device
+| Yes
+| 70 Years
+| Implements security functionality.
+
+| fusionauth.trust_c
+| No
+| 70 Years
+| Implements security functionality.
+
+| fusionauth.trust_t
+| No
+| 70 Years
+| Implements security functionality.
```

Comment:
This cookie represents a trust token used to complete an action that requires trust as provided by a step up authentication request.